rhcos.sh

```diff
@@ -0,0 +1,11 @@
+# Get the git commit that the openshift installer was built from
+OPENSHIFT_INSTALL_COMMIT=$($OPENSHIFT_INSTALLER version | grep commit | cut -d' ' -f4)
+
+# Get the rhcos.json for that commit, and find the baseURI and openstack image path
+RHCOS_IMAGE_JSON=$(curl https://raw.githubusercontent.com/openshift/installer/$OPENSHIFT_INSTALL_COMMIT/data/data/rhcos.json)
```

Comment:
We could export this in the case where we've already built the installer locally from source, then just do the curl as a default in the case where it's from the image?